ansible/roles/fews/files/glossis_workflow.yml

```diff
@@ -0,0 +1,121 @@
+kind: Workflow
+metadata:
+  generateName: glossis-
+spec:
+  entrypoint: complete-workflow
+  volumes:
+  - name: dgds-storage-cleanup
+    persistentVolumeClaim:
+      claimName: dgds2
+  - name: dgds-storage-waterlevel
+    persistentVolumeClaim:
+      claimName: dgds-waterlevel
+  - name: dgds-storage-currents
+    persistentVolumeClaim:
+      claimName: dgds-currents
+  - name: dgds-storage-wind
+    persistentVolumeClaim:
+      claimName: dgds-wind
+  - name: dgds-storage-waves
+    persistentVolumeClaim:
+      claimName: dgds-waves
+  - name: dgds-storage-flowmap
+    persistentVolumeClaim:
+      claimName: dgds-flowmap
+  - name: google-cred
+    secret:
+      secretName: serviceaccount
+
+  templates:
+  - name: complete-workflow
+    dag:
+      tasks:
+      - name: cleanup
+        template: glossis
+        # Argo arguments
+        arguments:
+          # Argo parameters
+          parameters:
+          # input argument for python script
+          - name: argument
+            value: cleanup
+          - name: volume
+            value: dgds-storage-cleanup
+      - name: waterlevel
+        dependencies: [cleanup]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: waterlevel
+          - name: volume
+            value: dgds-storage-waterlevel
+      - name: currents
+        dependencies: [cleanup]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: currents
+          - name: volume
+            value: dgds-storage-currents
+      - name: wind
+        dependencies: [cleanup]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: wind
+          - name: volume
+            value: dgds-storage-wind
+      - name: waves
+        dependencies: [cleanup]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: waves
+          - name: volume
+            value: dgds-storage-waves
+      - name: flowmap-tiffs
+        dependencies: [currents]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: flowmap-tiffs
+          - name: volume
+            value: dgds-storage-flowmap
+      - name: flowmap-tiles
+        dependencies: [flowmap-tiffs]
+        template: glossis
+        arguments:
+          parameters:
+          - name: argument
+            value: flowmap-tiles
+          - name: volume
+            value: dgds-storage-flowmap
+
+  - name: glossis
+    inputs:
+      parameters:
+      - name: argument
+      - name: volume
+    container:
+      imagePullPolicy: Always
+      image: deltares/dgds-glossis:test
+      command: ["python"]
+      args: ["/root/glossis.py", "dgds-data", "fews_glossis/", "projects/dgds-gee/glossis", "gee", "--{{inputs.parameters.argument}}"]
```

Comment:
This one has "gee" while gloffis has not?

Reply:
Gloffis is not finished, because I don't have any data for testing. I'll put this as a remark in de gloffis_workflow file. I added the new gee_bucket_folder argument in glossis code for testing.